lib/schema_migrator.rb

```diff
@@ -10,7 +10,7 @@ def initialize(index_name, cluster: Clusters.default_cluster, wait_between_task_
 
   def reindex
     index_group.current.with_lock do
-      response = Services.elasticsearch(cluster: cluster, timeout: 60).reindex(
+      response = Services.elasticsearch(hosts: 'http://localhost:9200?slices=auto', cluster: cluster, timeout: 60).reindex(
```

Comment:
The URL isn't `localhost:9200` in AWS, and the presence of the `cluster` argument means that the `hosts` argument [gets ignored anyway](https://github.com/alphagov/search-api/blob/2c92aecd8f557121f4e8d42d2a6e95902afa93ca/lib/services.rb#L31).  This should be:

```ruby
response = Services.elasticsearch(hosts: "#{cluster.uri}?slices=auto", timeout: 60).reindex(
```